src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
Another way to see that is: the tool chain of build got a libc or a kill implementation equivalent, and the tool to print the correct associated signal, so let the tool chain translate correctly the signal on any given platform and do not creates a <default> print that is not verify to be accurate.